x/batching/types/errors.go

```diff
@@ -0,0 +1,7 @@
+package types
+
+import "cosmossdk.io/errors"
+
+var (
+	ErrInvalidValidatorSetTrimPercent = errors.Register(ModuleName, 2, "validator set trim percent must be between 0 and 100")
```

Comment:
Do we want a custom error for this or use the standard invalid value one?

Reply:
I think I like the more specific ones more as it clarifies what the problem is?

Reply:
You can still add your own message to the standard invalid value error, so it won’t be complete guesswork as to what the invalid value was :)